### ISSUE TITLE
Organize files %artistinitial% uses Album Artist instead of Artist

### ISSUE THREAD

**Issue by jerobarraco** [clementine-player/Clementine#5780]
I'd like to be able to use the Artist tag instead of the Album Artist for %artistinitial% 
a) because is the logic way for me and makes sense for organizing
b) because the variable name %artistinitial% seems to indicate so.

Or maybe a better solution would be to have a %albumartistinitial% variable

**Comment by santigl** [comment]
I agree, we should make artists, albumArtists, and effectiveAlbumArtists consistent. 

@hatstand: Could we make those changes, considering that it might break people's formats?
